contrib/ovirt/lib/ovirtlago/__init__.py

```diff
@@ -524,7 +524,7 @@ def collect_artifacts(self, output_dir):
         os.makedirs(output_dir)
 
         def _collect_artifacts(vm):
-            with LogTask('%s' % vm.name()):
+            with LogTask('Collecting artifacts from VM %s' % vm.name()):
```

Comment:
I see no wrong in that phrase, why does it sound wrong to you?

David Caro

El 28/2/2016 12:29, Yaniv Kaul <notifications@github.com> escribió:

In contrib/ovirt/lib/ovirtlago/__init__.py:

> @@ -524,7 +524,7 @@ def collect_artifacts(self, output_dir): > os.makedirs(output_dir) > > def _collect_artifacts(vm): > - with LogTask('%s' % vm.name()): > + with LogTask('Collecting artifacts from VM %s' % vm.name()): 

Perhaps, but in English 'Collect artifacts from ..' sounds wrong.

—
Reply to this email directly or view it on GitHub.